api/models/config.go

```diff
@@ -125,3 +127,67 @@ func (h *Headers) Scan(value interface{}) error {
 	// otherwise, return an error
 	return fmt.Errorf("headers invalid db format: %T %T value, err: %v", value, bv, err)
 }
+
+// MilliCPU units
+type MilliCPUs uint64
+
+const (
+	MinMilliCPUs = 0       // 0 is unlimited
+	MaxMilliCPUs = 1024000 // 1024 CPUs
+)
+
+// implements fmt.Stringer
+func (c MilliCPUs) String() string {
+	if c == 0 {
+		return ""
+	}
+	return fmt.Sprintf("%dm", c)
+}
+
+// implements json.Unmarshaler
+func (c *MilliCPUs) UnmarshalJSON(data []byte) error {
+
+	outer := strings.TrimSpace(string(data))
```

Comment:
not a huge deal, but useful to know, golang `strings` and `bytes` package have nearly identical helper functions for trim/suffix/prefix/etc work. https://golang.org/pkg/bytes/#TrimSpace